content/applications/essentials/contacts.rst

```diff
@@ -178,13 +178,64 @@ section, and other partner options, including :guilabel:`Partner Activation` and
 Review` configurations. These are **only** present when the *Enterprise Resellers* module is
 installed.
 
+To publish a list of all active partners on the website, a new webpage must be created first.
+Navigate to :menuselection:`Website app --> Site --> Pages` and click :guilabel:`New`. Hover over
+the :guilabel:`Blank Page` template, and click the :guilabel:`Use this template` button to create a
+new page.
+
+In the :guilabel:`New Page` pop-up window, enter `Partners` in the :guilabel:`Page Title` field.
+Then, click :guilabel:`Create`. In the :guilabel:`Edit` toolbar on the far-right of the page, click
+:guilabel:`Save`.
+
+Return to :menuselection:`CRM app --> Sales --> Customers` and click the Kanban card for a partner.
+Click the :guilabel:`Go to Website` smart button at the top of the page to open the partner's
+webpage.
+
+Click :guilabel:`Edit` at the top-right of the page, and use the :doc:`building blocks
+<../websites/website/web_design/building_blocks>` to add any additional information about the
+partner.
+
+.. tip::
+  A company summary is a useful addition to this page.
+
+After making any necessary changes to the page, click :guilabel:`Save`. At the top of the page,
+slide the :guilabel:`Unpublished` toggle to the active, :guilabel:`Published` position.
+
+Repeat these steps for all partners.
+
 Membership tab
 --------------
 
 Finally, there is the :guilabel:`Membership` tab, which can help users manage any memberships that
 are being offered to this specific contact. It should be noted that this tab only appears when the
 *Members* applications is installed.
 
+Activate membership
+~~~~~~~~~~~~~~~~~~~
+
+To activate a contact's membership, click :guilabel:`Buy Membership`. On the pop-up window, select
+a :guilabel:`Membership` from the drop-down menu, then click :guilabel:`Invoice Membership`.
+
+Alternatively, to offer a free membership, tick the :guilabel:`Free Member` checkbox.
+
+To publish a list of all active members on the website, a new webpage must be created first.
+Navigate to :menuselection:`Website app --> Site --> Pages` and click :guilabel:`New`. Hover over
```

Comment:
```suggestion
To create a new webpage, navigate to :menuselection:`Website app --> Site --> Pages`, and click :guilabel:`New`. Hover over
```
You can also opt to mention that other route I mentioned above here, as well -- if you'd like...